sequana_pipelines/mapper/mapper.rules

```diff
@@ -152,10 +151,15 @@ elif config['general']['mapper'] == "bowtie2":
             options=config["bowtie2"]["options"]
         threads:
             config["bowtie2"]["threads"]
+        resources:
+            **config["bowtie2"]["resources"]
         container:
             config['apptainers']['bowtie2']
         wrapper:
             f"{manager.wrappers}/wrappers/bowtie2/align"
+else:
+    mapper = config['general']['mapper']
+    raise ValueError(f"mapper tool must be set to bwa, bowtie2 or minimap2. You provided {mapper}")
```

Comment:
Maybe this can be replaced by reusing the schema on the config file ?

Reply:
probably. We need additional tools in sequana_pipetools. I added an issue

Reply:
https://github.com/sequana/sequana_pipetools/issues/88